crt_portal/cts_forms/models.py

```diff
@@ -226,6 +226,16 @@ def get_absolute_url(self):
         return f'/form/view?{self.query}'
 
 
+class RetentionSchedule(models.Model):
+    name = models.CharField(max_length=255, null=False, blank=False, help_text="The name of the schedule that will be shown to intake specialists in dropdowns.")
+    order = models.IntegerField(default=0, help_text="The order in which to show the schedules, lower numbers first. If two schedules have the same order number, they might change positions.")
+    description = models.TextField(max_length=7000, null=False, blank=True, help_text="Internal notes, shown only here, about this schedule.")
+    tooltip = models.CharField(max_length=255, null=False, blank=True, help_text="The text to show in the tooltip for this schedule.")
+    retention_years = models.IntegerField(null=False, blank=False, help_text="The number of years to retain reports with this schedule (following their closure). Set to 0 for permanent retention.")
```

Comment:
Should we have a date field for the expiration date? Also wondering if we should have a field for whether the retention period is approved by the records team - or should we add that later?

Reply:
> Should we have a date field for the expiration date

I don't think so - these objects represent the schedules themselves (1 year, 10 year, etc) - they _don't_ represent the relationship of the schedule with a report

I wouldn't want to set an expiration date on these programmatically, either; we may lapse on the expiration date because of some policy extension, etc, where I wouldn't want to disable choosing the schedules automatically